termennetwerk.ts

```diff
@@ -51,22 +52,33 @@ export async function getUriOfSearchTerm(sources: string[], searchTerm: string):
     const cacheKey = searchTerm.toLowerCase();
     if (cache[cacheKey]) return cache[cacheKey];
     const response: GraphQlResponse = await client.request(graphQlQuery, { sources, query: searchTerm });
-    if (!Array.isArray(response.terms) ||
-        response.terms[0].result.terms.length !== 1 ||
-        !response.terms[0].result ||
-        !response.terms[0].result.terms ||
-        !Array.isArray(response.terms[0].result.terms)) {
+    if (!Array.isArray(response.terms[0].result.terms) ||   // terms[] does not exists when an error occurs like a timeout
+        response.terms[0].result.terms.length !== 1 ) {     // allow only a single result
+          if (response.terms[0].result.message) {
+            console.error('Error message from Network-of-Terms API: ',response.terms[0].result.message,' for keyword: ', searchTerm);
+          }
           return undefined;
     }
-
     /*
       searchTerm 'fietsen (transportmiddelen)' => http://vocab.getty.edu/aat/300212636 (prefLabel)
       searchTerm 'rijwiel' => http://vocab.getty.edu/aat/300212636 (altLabel)
       searchTerm 'rijwielen' => http://vocab.getty.edu/aat/300212636 (altLabel)
```

Comment:
These three examples presumably worked before, but now they yield an `undefined`: there are 2 or more AAT terms that match these search words. Perhaps add some new, working examples?

Reply:
I removed the examples for now, maybe add a csv with examples for testing sometime? ;)